jigar/hangman/hangman.rb

```diff
@@ -0,0 +1,45 @@
+$secret_word = "aaabbbcccc"

```

Comment:
we can use constant variable to declare constants

Reply:
Thanks for suggestion @Hardik-Jade. I have made changes